clients/mssql/queries.go

```diff
@@ -0,0 +1,34 @@
+package mssql
+
+import (
+	"fmt"
+
+	"github.com/artie-labs/transfer/lib/config/constants"
+)
+
+func describeTableQuery(schema, rawTableName string) string {
+	return fmt.Sprintf(`
+SELECT 
+    COLUMN_NAME, 
+    CASE
+        WHEN DATA_TYPE = 'numeric' THEN
+		'numeric(' + COALESCE(CAST(NUMERIC_PRECISION AS VARCHAR), '') + ',' + COALESCE(CAST(NUMERIC_SCALE AS VARCHAR), '') + ')'
+		ELSE
+		DATA_TYPE
+	END AS DATA_TYPE,
+    CHARACTER_MAXIMUM_LENGTH
+FROM 
+    INFORMATION_SCHEMA.COLUMNS
+WHERE 
+    LOWER(TABLE_NAME) = LOWER('%s') AND LOWER(TABLE_SCHEMA) = LOWER('%s');`, rawTableName, schema)
```

Comment:
Can we escape the table name and schema in case they have quotes in them?

Reply:
Param query is very hard because it's sql driver specific and this is in a shared function

Reply:
So we'll want all the functions to use it together.